egs2/librimix/enh1/local/data.sh

```diff
@@ -150,4 +150,46 @@ if [ ${stage} -le 3 ] && [ ${stop_stage} -ge 3 ]; then
     done
 fi
 
+if [ ${stage} -le 4 ] && [ ${stop_stage} -ge 4 ]; then
+    log "stage 4: LibriMix Transcripts Files Prepration."
+
+    mkdir -p data/local
+    for dset in "train-clean-100" "train-clean-360" "dev-clean" "test-clean"; do
+        for reader_dir in $(find -L "${LIBRISPEECH}/${dset}" -mindepth 1 -maxdepth 1 -type d | sort); do
```

Comment:
can we simply reuse `librispeech/asr1/local/data_prep.sh` here?

Reply:
I try to minimize the data needed for the data preparation here. For this data, we only need the clean data to prepare the transcript here.

Reply:
I think it is called on every single set.
```
if [ ${stage} -le 2 ] && [ ${stop_stage} -ge 2 ]; then
    log "stage 2: Data Preparation"
    for part in dev-clean test-clean dev-other test-other train-clean-100 train-clean-360 train-other-500; do
        # use underscore-separated names in data directories.
        local/data_prep.sh ${LIBRISPEECH}/LibriSpeech/${part} data/${part//-/_}
    done
fi
```

Reply:
OK. Now I reused it.